lib/plugins/aws/package/compile/events/cognitoUserPool/index.js

```diff
@@ -350,7 +350,10 @@ class AwsCompileCognitoUserPoolEvents {
       const userPoolLogicalId = this.provider.naming.getCognitoUserPoolLogicalId(poolName);
 
       // If overrides exist in `Resources`, merge them in
-      if (_.has(this.serverless.service.resources, userPoolLogicalId)) {
+      if (
+        this.serverless.service.resources &&
+        this.serverless.service.resources[userPoolLogicalId]
```

Comment:
I think it can be just `if (this.serverless.service.resources[userPoolLogicalId])` (?)